### ISSUE TITLE
Docs: add travis ci badge

### ISSUE THREAD

**Issue by AndreVarandas** [AndreVarandas/the-flappy-bird#19]
We want to display the status of master branch build in the readme.md file.

It should be included right after the first heading in the readme file:

```markdown
[![Build` `Status](https://travis-ci.com/AndreVarandas/the-flappy-bird.svg?branch=master)](https://travis-ci.com/AndreVarandas/the-flappy-bird)
```

**Comment by bsshashank** [comment]
I would like to work on this ticket.

**Comment by AndreVarandas** [comment]
Hi @bsshashank I’m assigning it to you, thanks!

**Comment by AndreVarandas** [comment]
Closed with #21
